docs/_utils/deploy.sh

```diff
@@ -2,7 +2,7 @@
 
 # Copy contents
 mkdir gh-pages
-cp -r ./docs/_build/dirhtml/* gh-pages
+cp -r ./docs/_build/dirhtml/. gh-pages
```

Comment:
@dgarcia360 what this change for ?

Reply:
To include all the files, including hidden files like ``.nojekyll``.

Reply:
The latest version of the theme generates the .nojekyll file inside ``./docs/_build/dirhtml``.